### PR TITLE
bpo-45639: Add webp and avif image formats to mimetypes

### PR DESCRIPTION
**Pull request by kixorz**
Modern image types webp and avif are not recognized by the mimetypes module.

Problem: Many tools are written in Python and running on macOS. Good example is the AWS CLI. Running commands like "s3 sync" will save files with .webp and .avif extensions with incorrect "binary/octet-stream" Content-Type to S3. This creates additional problems with serving these resources over HTTP.

The webp and avif image types are supported by most browsers:
https://caniuse.com/#feat=webp
https://caniuse.com/#feat=avif

While webp is fully supported and largely used, it is not officially registered with IANA.

Avif is currently less popular, but it is fully registered with IANA.
https://www.iana.org/assignments/media-types/media-types.xhtml

Please consider the attached GitHub PR as a fix to these MIME Content-Type issues.

https://bugs.python.org/issue45639

#89802

### CODE REVIEW

**Comment by merwok** [comment]
wepb is already added in #23034 (and with proper alpha sorting in the list)

**Comment by kixorz** [comment]
I saw the other PR, but it's not merged yet. The addition in the commit in this PR is sorted by mime type as requested in comment in the file.

**Review by MaxwellDupre — CHANGES_REQUESTED** [review]
Agreed these are not new and should be added but could you add to test Lib/test/test_mimetypes.py please?

**Comment by kixorz** [comment]
Thank you. I'll add the test.

**Comment by merwok** [comment]
On second thoughts, one ticket and one PR to add both types is fine.

**Comment by kixorz** [comment]
I added the test.

**Comment by merwok** [comment]
Can you add a news note on this model: https://github.com/python/cpython/pull/24917/files#diff-068155607fe32979f71408456e0f3163740897c21e6b6da80c45ec59cc660424

You can use https://blurb-it.herokuapp.com/ or `pip install blurb` locally

**Comment by TeamSpen210** [comment]
If `webp` is nonstandard, shouldn't it go in the non-strict dict instead?

**Comment by kixorz** [comment]
I think I added the news entry. Thanks for the guidance.

**Review by JelleZijlstra — CHANGES_REQUESTED** [review]
Thanks, and sorry this was left hanging for so long! We still have a few days until the 3.11 feature freeze, and I'd love to get this in.

A few small things:
- You'll need to resign the CLA with our new process
- As @TeamSpen210 says, webp is nonstandard and should be in the `common_types` list.
- The docs build is failing because the blurb is missing a trailing newline.

**Comment by kixorz** [comment]
I have made the requested changes; please review again.

**Review by JelleZijlstra — APPROVED** [review]

**Comment by JelleZijlstra** [comment]
@kixorz thanks for the change! The tests are failing now, I think webp needs to move elsewhere in the test suite because it's not recognized by default.

**Comment by kixorz** [comment]
I believe I fixed the tests.

File: Lib/test/test_mimetypes.py [hunk 1]
```diff
@@ -95,13 +95,16 @@ def test_non_standard_types(self):
         eq = self.assertEqual
         # First try strict
         eq(self.db.guess_type('foo.xul', strict=True), (None, None))
+        eq(self.db.guess_type('foo.xul', strict=True), (None, None))
```
Comment (JelleZijlstra) [inline Lib/test/test_mimetypes.py]: This is duplicated
```suggestion
```

**Comment by kixorz** [comment]
Thanks for checking it. Sorry, I missed it. Should be fine now.

**Review by JelleZijlstra — APPROVED** [review]

**Comment by kixorz** [comment]
Thanks for merging my changes!

**Pull request merged**